src/routes/api/[...endpoint]/+server.ts

```diff
@@ -1,10 +1,6 @@
 import type { RequestHandler } from '@sveltejs/kit';
 import { fetchWithAuth } from './proxy';
-
-function endpoint(url: URL) {
-  const pathName = url.pathname.replace('/api', '');
-  return pathName.startsWith('/') ? pathName.substring(1) : pathName;
-}
+import { endpoint } from '../../../lib/api/proxy';
```

Comment:
Maybe avoid using relative imports, if possible.

Reply:
I didn't find a rule for that, unfortunately :/